scripts/host-check

```diff
@@ -1169,6 +1182,90 @@ def check_shmem_pages_max_size() -> CheckFuncReturn:
     return CheckState.SUCCESS, f"{shared_mem_max_page_size:.1f} GiB"
 
 
+def _module_param_compare(
```

Comment:
Rename to `_module_params_compare` - i.e. plural 'params', as this function is checking all of them for the module.